projects/kit/abstract/input-slider.ts

```diff
@@ -125,6 +126,9 @@ export abstract class AbstractTuiInputSlider<T>
     mask: TuiMapper<number, TuiTextMaskOptions> = (quantum: number, min: number) => ({
         mask: tuiCreateNumberMask({
             allowNegative: min < 0,
+            autoCorrectDecimalSymbol: !TUI_DECIMAL_SYMBOLS.includes(
```

Comment:
nit: there is todo in tuiCreateNumberMask about deleting autocorrectDecimalSymbol in 3.0, so if it makes sense now can you delete todo plz?

Reply:
fixed